simulariumio/orientations/__init__.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from .particle_rotation_calculator import ParticleRotationCalculator  # noqa: F401
+from .orientation_data import OrientationData  # noqa: F401
+from .neighbor_data import NeighborData  # noqa: F401
+from .rotation_utility import RotationUtility  # noqa: F401
```

Comment:
IMO, everything in this directory should really be a subdirectory of `readdy` (except maybe rotation_utility).  I'd just move the other 3 files into `readdy/orientations`.  All the data structures are so custom to readdy for now.

Also wondering if there is code to generate OrientationData from pdb file.  That code would be great to have in this repo really well.

Reply:
there's a procedure I use in chimera, I can add it to the documentation at least. It's all things that could be a script but aren't yet.